### DISCUSSION TITLE
Impersonation mode

### DISCUSSION THREAD

**Discussion by Jampire** [moonshine-software/moonshine#293] [General]
Реализовал пакет для работы с подменой пользователя: [MoonShine Impersonate](https://github.com/Jampire/moonshine-impersonate).

- [Документация](https://dzianiskotau.com/moonshine-impersonate/).

Что такое подмена пользователя в других решениях:

- [Laravel Nova impersonation mode](https://nova.laravel.com/docs/4.0/customization/impersonation.html)
- [Laravel Orchid impersonation mode](https://orchid.software/en/docs/access/#user-impersonation)
- [Laravel standalone package](https://laravel-news.com/laravel-impersonate)

**Comment by lee-to** [comment]
Здравствуйте! Спасибо за пакет, мы добавим его в документацию в раздел пакеты в ближайшее время
Также будем рады если вы в репозитории создадите директорию art где будут скриншоты реализации (необходимо для будущего  маркета плагинов для муншайн)

**Reply by Jampire** [reply]
Добавил скриншоты.

**Comment by lee-to** [comment]
https://moonshine.cutcode.dev/section/packages

**Comment by Jampire** [comment]
В версии [1.1.0](https://github.com/Jampire/moonshine-impersonate/releases/tag/v1.1.0) добавлена возможность записи действий пользователя в журнал MoonShine. Подробности [тут](https://dzianiskotau.com/moonshine-impersonate/#usage_logs).

@lee-to Предлагаю открыть эту тему. Или добавить где-нибудь раздел, где авторы расширений могут публиковать свои анонсы.
